### PR TITLE
Update adapter creation guide

### PR DESCRIPTION
**Pull request by Liberatys**
@coorasse I reopened this pr as there was an issue with travis. 
Should pass now

### CODE REVIEW

**Comment by tagliala** [comment]
There are some failures because activerecord-edge now requires Ruby >= 2.7

**Review by coorasse — APPROVED** [review]

File: docs/Model-Adapter.md [hunk 1]
```diff
@@ -126,6 +195,27 @@ def self.matches_conditions_hash?(subject, conditions)
...
+
+* [ActiveRecord Base](https://github.com/CanCanCommunity/cancancan/blob/develop/lib/cancan/model_adapters/active_record_adapter.rb)
+* [ActiveRecord 4](https://github.com/CanCanCommunity/cancancan/blob/develop/lib/cancan/model_adapters/active_record_4_adapter.rb)
+* [ActiveRecord 5](https://github.com/CanCanCommunity/cancancan/blob/develop/lib/cancan/model_adapters/active_record_5_adapter.rb)
+
+#### Tests / Specs
+
+* [ActiveRecord Base](https://github.com/CanCanCommunity/cancancan/blob/develop/spec/cancan/model_adapters/active_record_4_adapter_spec.rb)
+* [ActiveRecord 4](https://github.com/CanCanCommunity/cancancan/blob/develop/spec/cancan/model_adapters/active_record_5_adapter_spec.rb)
+* [ActiveRecord 5](https://github.com/CanCanCommunity/cancancan/blob/develop/spec/cancan/model_adapters/active_record_adapter_spec.rb)
```
Comment (coorasse) [inline docs/Model-Adapter.md:218]: @Liberatys can you point to  `[ActiveRecord Base](./active_record_4_adapter_spec.rb)` for example?

Comment (Liberatys) [reply docs/Model-Adapter.md:218]: 👍🏻

**Comment by Liberatys** [comment]
Any more adaptions needed?

**Comment by coorasse** [comment]
@Liberatys can I kindly ask you to rebase once more? The file changed name, is now `model_adapter.md`. Thanks 🙏

**Comment by Liberatys** [comment]
Sure thing !

**Pull request merged**